build.gradle

```diff
@@ -52,6 +52,12 @@ allprojects {
     apply plugin: 'idea'
     apply plugin: "io.spring.dependency-management"
 
+    idea {
```

Comment:
This will download Javadoc for external libraries during development. It will not affect anything else but provide developers with the documentation of the method they use.